src/object.d

```diff
@@ -1737,7 +1728,7 @@ class Throwable : Object
                     sink("\n"); sink(t);
                 }
             }
-            catch (Throwable)
+            catch (Exception)
             {
```

Comment:
Yeah I know, but I was trying to say that silently swallowing them as it is done in the existing code isn't really a good practice.